src/sdk/acceptAuthenticatedQuote.ts

```diff
@@ -0,0 +1,36 @@
+import {
+    type AcceptedQuote, type LiquidityProvider, quoteDetailRequiredFields,
+    quoteRequiredFields, type Quote, Routes, providerRequiredFields
+  } from '../api'
+  import { type LiquidityBridgeContract } from '../blockchain/lbc'
+  import { FlyoverError } from '../client/httpClient'
+  import { type HttpClient, isValidSignature, validateRequiredFields } from '@rsksmart/bridges-core-sdk'
+
+  export async function acceptAuthenticatedQuote (httpClient: HttpClient, lbc: LiquidityBridgeContract,
+    provider: LiquidityProvider, quote: Quote, signature: string): Promise<AcceptedQuote> {
+    validateRequiredFields(quote, ...quoteRequiredFields)
+    validateRequiredFields(quote.quote, ...quoteDetailRequiredFields)
+    validateRequiredFields(provider, ...providerRequiredFields)
+
+    const url = provider.apiBaseUrl + Routes.acceptQuoteFromTrustedAccount
+    const acceptedQuote = await httpClient.post<AcceptedQuote>(
+      url,
+      { QuoteHash: quote.quoteHash, Signature: signature },
```

Comment:
The QuoteHash with capital Q is a legacy error that hasn't been fixed because of compatibility reasons, if you're adding Signature I'd add it as signature from the beginnig

Reply:
I'll fix this in the next PR coming with the changes for pegout.